en-us/api/Microsoft_Toolkit_Uwp_UI_CacheBase.md

```diff
@@ -0,0 +1,101 @@
+---
+permalink: /en-US/api/Microsoft_Toolkit_Uwp_UI_CacheBase.htm
+title: Microsoft.Toolkit.Uwp.UI.CacheBase API 
+description: API page for Microsoft.Toolkit.Uwp.UI.CacheBase
+keywords: windows, app, toolkit, UWP, API
+layout: api
+search.product: eADQiWindows 10XVcnh
+---
+
+
+# CacheBase<T> class
+
+Represents a strongly typed cache of objects that can be added to and removed as desired.
+The cache also provides in-memory storage of strongly typed objects to reduce reading from disc. The size of this in-memory storage is controlled by setting the ***MaxMemoryCacheCount*** property.
+The in-memory storage follows FIFO (first in first out) and the caller can pass optional parameter when calling ***PreCacheAsync*** to inform cache to use in-memory storage.
+
+## Members
+
+The **CacheBase<T>** class has the following types of members:
+
+### Methods
+
+#### InitializeAsync(Windows.Storage.StorageFolder folder, System.String folderName)
+
+this method is used supply a folder within which cache specific folder is created based on name provided.
```

Comment:
"This method is used to supply a StorageFolder and a cache folder name where the cache items will be stored"